controllers/cache_handler.go

```diff
@@ -243,7 +243,11 @@ func (h DisruptionTargetWatcherHandler) findNotifiableEvents(eventsToSend map[st
 					"message", event.Message,
 					"timestamp", event.LastTimestamp.Time.Unix(),
 				)
-			} else if event.Reason == "Started" {
+
+				continue
+			}
+
+			if event.Reason == "Started" {
```

Comment:
`else if event.Reason == "Started"` should work fine here, no?

Reply:
I responded to the comments from @ptnapoleon